sys/net/gnrc/network_layer/ipv6/nib/nib.c

```diff
@@ -1312,6 +1312,7 @@ static bool _resolve_addr(const ipv6_addr_t *dst, gnrc_netif_t *netif,
             entry = _nib_nc_add(dst, (netif != NULL) ? netif->pid : 0,
                                 GNRC_IPV6_NIB_NC_INFO_NUD_STATE_INCOMPLETE);
             if (entry == NULL) {
+                DEBUG("nib: can't resolve address, nib full\n");
```

Comment:
```suggestion
                DEBUG("nib: can't resolve address, NC full\n");
```

when the NIB is full, neighbor cache entries for the same address and interface should still be able to be created, so let's be more factual correct ;-)